docs/general/polkadot-v1.md

```diff
@@ -0,0 +1,154 @@
+  - [The Polkadot Blockchain Academy](https://polkadot.network/development/academy/)
+  - [Substrate Builders Program](./builders-program.md)
+  - [Polkadot Developer Heroes Program](./dev-heroes.md)
+  - [Edx Courses](https://www.edx.org/school/web3x)
+  - Rust and Substrate Courses (coming soon)
+
+### Polkadot's Representation
+
+The Polkadot realy-chain has been represented as a relay chain surrounded by multiple parachains
+attached to it. Parachains can communicate with each other using a generalized
+[cross-chain messaging format (XCM)](./learn-xcm-index) allowing messages to be delivered across
+chains having different logic.
+
+### Polkadot's Additional Functionalities
+
+The Polkadot relay-chain also manges [crowdloans](../learn/learn-crowdloans.md),
+[auctions](../learn/learn-auction.md), [staking](../learn/learn-staking.md),
+[accounts](./learn-accounts-index), [balances](../learn/learn-balance-transfers.md), and
+[governance](../learn/learn-polkadot-opengov.md). Parachain slots or cores are leased in 6-month
+chunks for a maximum of two years, and crowdloans allow users to trustlessly loan funds to teams for
+lease deposits in exchange for pre-sale tokens. There is no other way you could use Polkadot 1.0.
+
+### Polkadot's Resilience
+
+Decentralization is a crucial aspect of blockchain networks, but there is a trade-off between:
+
+- having an over-decentralized network that struggles to reach consensus and consumes a lot of
+  energy to operate, and
+- having a network that reaches consensus fast at the expense of being centralized, making it
+  trivial to manipulate or attack.
+
+Ideally, a network should be decentralized "enough" to make it practically impossible for someone to
+exert manipulative or malicious influence on the network. So, **decentralization is a tool while the
+goal is resilience**, which is achieved by additionally providing on-chain treasury and governance
+mechanism allowing continuous incentives for the network's participants without relying on
+intermediaries or centralized entities.
+
+Currently, Polkadot 1.0 achieve resilience through the following strategies:
+
+- Nominated Proof of Staking (NPoS) where the stake per validator is maximized and evenly
+  distributed across validators.
+- The [1KV program](./thousand-validators.md) aims to incentivize new operators to become network
+  participants and further increase physical (how many validator nodes per service provider) and
+  social decentralization (how many validator nodes per operator). Those can be explored with the
+  [Polkawatch App](https://polkadot.polkawatch.app/).
+- An on-chain treasury and governance where every decision goes though public referenda and any
+  token holder can cast a vote.
+
+### Polkadot's Blockspace
+
+The design and realization of Polkadot 1.0 allowed its creators to truly realize the importance of
+blockspace.
+
+A blockchain is a way to store data. The storage unit is the block, and once a block is finalized
```

Comment:
```suggestion
Blockspace represents the raw material that is produced as a result of the secure, trustless compute that blockchains provide.  A blockchain defines a way to propagate a set of state changes, where the storage unit for these state changes is the block, and once a block is finalized
```

Reply:
Blockspace can be non-secure and non-trustless. So I do not fully agree that blockspace "is produced as a result of secure and trustless computation". Do you also agree? I can produce a block with some data myself without a blockchain.

Reply:
Of course, but if we're talking about Polkadot's specific goal, then Polkadot provides that.  I can edit it to reflect that particular point